docs/content.md

```diff
@@ -470,6 +471,67 @@ content:
 ---
 ```
 
+## Creating an inspiration page
+
+Create a folder under ‘content/life-as-a-teacher/sub-category’
+
+From a previous published page copy the files:
+
+_article.html/erb
+_header.html.erb
+
+Also copy and paste a previous ‘.md’ file and rename it with the title (e.g abigails-career-progression-story.md)
```

Comment:
Do we need to specify that the .md file should live in the subcategory folder, but not within the same folder as the article/header